src/models/Journal.js

```diff
@@ -1,22 +1,54 @@
-import mongoose from "mongoose";
+import mongoose, { Schema } from "mongoose";
 
-const JournalSchema = new mongoose.Schema({
+const journalSchema = new mongoose.Schema({
   title: {
     type: String,
     required: true,
   },
+
   url: {
     type: String,
     required: true,
   },
+
   issn: {
     type: Number,
     required: true,
+    unique: true,
   },
-  domain: {
+
+  domainName: {
     type: String,
-    required: true,
+    required: false,
+  },
+
+  policies: {
```

Comment:
Hey, @d-e-v-esh we had also planned to have a `policyType` field right?

Reply:
Shall I rename `policies` to `polityType`?